config/samples/import_plugins/image_conversion/image_conversion.yaml

```diff
@@ -0,0 +1,56 @@
+# Sample using Ceph as a glance backend with image conversion plugin
+# Requires a running Ceph cluster and its `/etc/ceph` files in secret `ceph-conf-files`
+# This can be achieved with the `ceph` target of `install_yamls`
+apiVersion: glance.openstack.org/v1beta1
+kind: Glance
+metadata:
+  name: glance
+spec:
+  serviceUser: glance
+  containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+  customServiceConfig: |
+    [DEFAULT]
+    enabled_backends = default_backend:rbd
+    [glance_store]
+    default_backend = default_backend
+    [default_backend]
+    rbd_store_ceph_conf = /etc/ceph/ceph.conf
+    store_description = "RBD backend"
+    rbd_store_pool = images
+    rbd_store_user = openstack
+    [image_import_opts]
+    image_import_plugins = ['image_conversion']
+    [image_conversion]
+    output_format = raw
+  databaseInstance: openstack
+  databaseUser: glance
+  glanceAPIInternal:
+    debug:
+      service: false
+    preserveJobs: false
+    replicas: 1
+  glanceAPIExternal:
+    debug:
+      service: false
+    preserveJobs: false
+    replicas: 1
+  secret: osp-secret
+  storageClass: ""
+  storageRequest: 1G
+  extraMounts:
```

Comment:
I think here we're missing the path where the pvc is mounted

Reply:
Could you please point out/add exact code snippet ?

Reply:
```
  extraMounts:
    - name: v1
      region: r1
      extraVol:
        - propagation:
          - Glance
          extraVolType: Multibackends
          volumes:
          - name: ceph
            projected:
              sources:
              - secret:
                  name: ceph-conf-files
          - name: imageconv
            persistentVolumeClaim:
              claimName: imageconv
              readOnly: false
          mounts:
          - name: ceph
            mountPath: "/etc/ceph"
            readOnly: true
          - name: imageconv
            mountPath: /var/lib/glance/os_glance_staging_store/
            readOnly: false
```

Reply:
Thank You!